src/components/AccountCard/index.tsx

```diff
@@ -13,7 +13,7 @@ export interface AccountCardPropType {
   categories?: number[];
 }
 
-const DESCRIPTION_MAX_LENGTH = 50;
```

Comment:
Exporting the same thing twice in AccountCard/index.tsx and SensorCard/index.tsx
Would it make sense to have this in one place or are these lengths bound to visual space that is different on each card?

Reply:
Hmm, even though they have the same limit now, I would still not couple the values here.

The elements and the space on the cards are different, so it is possible that we need to separate them again when we touch the appearance of the cards. IMO, there is enough visual and semantic difference to the cards that justifies keeping the separation of the variables.